workflow_scripts/tests/test_mutation_run_mutmut.py

```
@@ -188,6 +223,51 @@ def test_failing_baseline_propagates_exit_code(
             "a failing mutmut run should propagate its exit code"
         )
 
+    @POSIX_SHIMS_ONLY
+    def test_workflow_input_env_does_not_leak_into_mutmut(
+        self, tmp_path: Path, monkeypatch: pytest.MonkeyPatch, fake_uv: Path
```

Ruff (0.15.21)

[warning] Unused method argument: `fake_uv`

(ARG002)